src/sentry/buffer/__init__.py

```diff
@@ -6,3 +8,18 @@
 
 backend = LazyServiceWrapper(Buffer, settings.SENTRY_BUFFER, settings.SENTRY_BUFFER_OPTIONS)
 backend.expose(locals())
+
+if TYPE_CHECKING:
+    __buffer__ = Buffer()
+    get = __buffer__.get
+    incr = __buffer__.incr
+    process = __buffer__.process
+    process_pending = __buffer__.process_pending
+    process_batch = __buffer__.process_batch
+    validate = __buffer__.validate
+    push_to_sorted_set = __buffer__.push_to_sorted_set
+    push_to_hash = __buffer__.push_to_hash
+    get_sorted_set = __buffer__.get_sorted_set
+    get_hash = __buffer__.get_hash
+    delete_hash = __buffer__.delete_hash
+    delete_key = __buffer__.delete_key
```

Comment:
this shouldn't be done like this -- the code should use the `backend` instance and access attributes off of that

the eventual plan is to eliminate `expose(...)` entirely

Reply:
Ahh, I wasn't aware of this. So the plan going forward is to use `sentry.buffer.backend.<attrs>`? I'm ok with that, but it might be worth letting people know

Reply:
99% of the usages have been moved over and the type checker tends to enforce it in new modules already